Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -9,11 +9,11 @@
 import Foundation
 
 public protocol LintableFileManager {
-    func filesToLintAtPath(_ path: String, rootDirectory: String?) -> [String]
+    func filesToLint(inPath: String, rootDirectory: String?) -> [String]
```

Comment:
if the parameter was a `URL` type, that'd be more obvious. But as it stands, the string could be anything, so the API specifies what the string is used for.